versioned_docs/version-8.6/guides/react-components/install-c8run.md

```diff
@@ -1,7 +1,7 @@
 ### Prerequisites
 
-:::warning
-Due to a bug in 8.6, cookie authentication is required to use the Tasklist, Operate, and Zeebe APIs included in Camunda 8 Run. For an example of authenticating via cookie, see the [Operate API documentation](/apis-tools/operate-api//authentication.md#authentication-via-cookie-self-managed-only).
+:::caution
```

Comment:
I think the Camunda REST API is not referenced or promoted in the guides, so why is this "caution" necessary?